src/main/groovy/de/dkfz/roddy/BEIntegrationTestStarter.groovy

```diff
@@ -128,7 +128,7 @@ class BEIntegrationTestStarter {
             // The job abortion might be a valid command but the cluster system might still try to keep the jobs.
             // We cannot handle problems like a stuck cluster, so let's stick to the basic query and see, if the job
             // ends within some seconds.
-            ensureProperJobStates(maxSleep, jobList, [JobState.ABORTED, JobState.OK, JobState.COMPLETED_UNKNOWN, JobState.COMPLETED_UNKNOWN], jobManager)
+            ensureProperJobStates(maxSleep, jobList, [JobState.ABORTED, JobState.COMPLETED_UNKNOWN, JobState.COMPLETED_UNKNOWN], jobManager)
```

Comment:
Why did you remove OK? You can also remove the double UNKNOWN. Only one is needed.